frontend/src/components/Login.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
+    let active = false;
+
+    function isActive() {
+        return active ? "is-active" : "";
+    }
+</script>
+
+<style>
+    .modal-content {
+        width: 31.25em;
+        height: 35.5em;
+        background: #D9D9D9;
+        border-radius: 7px;
+    }
+
+    .link:hover {
+        text-decoration: underline;
+        color: #276BB0;
+    }
+
+    #login-contents {
+        margin: 2.5em 3.125em 0em;
+    }
+</style>
+
+<button class="button is-light" on:click={() => { active = true; }}>
+    <strong>Log in</strong>
+</button>
+
+{#if active}
+    <div class="modal {isActive()}">
+    <div class="modal-background" color=""/>
+    <div class="modal-content">
+        <div class="is-flex is-flex-direction-column">
+            <div class="is-align-self-flex-end">
+                <button class="delete is-large" style="background: #D9D9D9" aria-label="close" on:click={() => { active = false; }}></button>
+            </div>
+            <div class="is-flex is-flex-direction-column pt-4" id="login-contents">
+                <p class="title is-1 has-text-black">Log in</p>
+
+                <div class="field mt-5">
+                    <div class="control">
+                        <input class="input" type="email" placeholder="    Email" />
```

Comment:
Also stylistic suggestion:
`<input class="input" type="email" placeholder="Email" />`
Since when actually entering the field, there would be no indent at the front, it'd probably be more consistent to remove the indent here as well.

Reply:
I was thinking the same thing, but I was just trying to match the Figma design